beacon_node/network/src/network_beacon_processor/gossip_methods.rs

```diff
@@ -1849,29 +1956,30 @@ impl<T: BeaconChainTypes> NetworkBeaconProcessor<T> {
                                 &metrics::BEACON_PROCESSOR_UNAGGREGATED_ATTESTATION_REQUEUED_TOTAL,
                             );
                             let processor = self.clone();
-                            ReprocessQueueMessage::UnknownBlockUnaggregate(QueuedUnaggregate {
-                                beacon_block_root: *beacon_block_root,
-                                process_fn: Box::new(move || {
-                                    processor.process_gossip_attestation(
-                                        message_id,
-                                        peer_id,
-                                        attestation,
-                                        subnet_id,
-                                        should_import,
-                                        None, // Do not allow this attestation to be re-processed beyond this point.
-                                        seen_timestamp,
-                                    )
-                                }),
-                            })
+                            let msg =
+                                ReprocessQueueMessage::UnknownBlockUnaggregate(QueuedUnaggregate {
+                                    beacon_block_root: *beacon_block_root,
+                                    process_fn: Box::new(move || {
+                                        processor.process_gossip_attestation(
+                                            message_id,
+                                            peer_id,
+                                            attestation,
+                                            subnet_id,
+                                            should_import,
+                                            None, // Do not allow this attestation to be re-processed beyond this point.
+                                            seen_timestamp,
+                                        )
+                                    }),
+                                });
+
+                            if sender.try_send(msg).is_err() {
+                                error!(
+                                self.log,
+                                "Failed to send attestation for re-processing";
```

Comment:
formatting here is weird too, if we fix the overly long `// Do not allow..` comments above then rustfmt might come to its senses